grakn-graql/src/main/java/ai/grakn/graql/internal/reasoner/rule/InferenceRule.java

```diff
@@ -49,8 +42,8 @@
 
     public InferenceRule(Rule rule, GraknGraph graph){
         QueryBuilder qb = graph.graql();
-        body = new Query(qb.match(rule.getLHS()), graph);
-        head = new AtomicQuery(qb.match(rule.getRHS()), graph);
+        body = new Query(qb.setInference(false).match(rule.getLHS()), graph);
+        head = new AtomicQuery(qb.setInference(false).match(rule.getRHS()), graph);
     }
```

Comment:
this shouldn't have any effect, the input match queries are only used to form the atoms on construction and later discarded